VITAL/donations.yml

```diff
@@ -32,3 +32,18 @@
   files:
     - VITAL/lobby16/2021-06-25/PaymentConfirmation.pdf
 
+- ticker: VITAL
+  announcement_url: *vitalmission
+  recipient: &veritree
+    name: Veritree Cardano Forest
+    address: 230-1275 Venables Street Vancouver, BC V6A 2E4
+    tel: 1-833-520-1145
+    contact: info@veritree.com
+  ada: 500
+  usd: 1060
+  timestamp: 1635036960
+  files:
+    - VITAL/lobby16/2021-06-25/PaymentConfirmation.pdf
```

Comment:
This is the same file from the previous donation. If you could update this with the correct file the rest looks good.

You may also need to update your repo with the main branch to make sure it is not out of date. Instructions are in the [README.md](https://github.com/CardanoMDP/Donations#keeping-your-repository-up-to-date-with-the-upstream-repository-cardanomdpdonations)

Reply:
Removed the file as Veritree is purely documented in the Cardano Transaction.
Thanks for working on the PRs!